node/network/src/peer_manager.rs

```diff
@@ -60,7 +57,7 @@ impl<T: ChainStateRetriever> PeerManager<T> {
         // Spawn peers that represent boot nodes.
         Peer::spawn_from_known(
             node_info.clone(),
-            boot_nodes.to_vec(),
+            boot_nodes.to_vec().drain(..).map(std::convert::Into::into).collect(),
```

Comment:
`drain` and `into_iter` are somewhat redundant to each other, but when the drained collection is not planned to be used it makes sense to use `into_iter` instead.
```suggestion
            boot_nodes.to_vec().into_iter().map(std::convert::Into::into).collect(),
```

Reply:
Fixed.